core/codegen/tests/from_form.rs

```diff
@@ -618,3 +618,33 @@ fn test_nested_multi() {
 //     println!("String: {}", string);
 //     panic!(string);
 // }
+
+#[test]
+fn test_defaults() {
```

Comment:
Can we test more types? Ideally every "unique-ish" type we implement `FromForm` for (one or so per entry in https://api.rocket.rs/master/rocket/form/trait.FromForm.html#provided-implementations). The following should work, for example:

```rust
#[form(default = vec![1, 2, 3])]
numbers: Vec<usize>,
```